### PR TITLE
Removing unused file

### PR DESCRIPTION
**Pull request by OlegOrlovsky**
Fixing import issue
Handling pararmeters/values at Import from qTest

### CODE REVIEW

File: Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs [hunk 1]
```diff
@@ -219,151 +219,21 @@ public static BusinessFlow ConvertQtestTestSuiteToBF(QtestTestSuite testSuite)
...
-                            {
-                                stepActivityVar.LinkedVariableName = string.Empty; // clear old links
-                            }
-                        }
+                            stepActivityVar = new VariableSelectionList();
+                            stepActivityVar.Name = step.Params.Name;
+                            ObservableList<OptionalValue> optionalValuesList = new ObservableList<OptionalValue>();
+                            step.Params.Values.ForEach(z => optionalValuesList.Add(new OptionalValue(z)));
+                            ((VariableSelectionList)stepActivityVar).OptionalValuesList = optionalValuesList;
+                            ((VariableSelectionList)stepActivityVar).SetValue(step.Params.Value);
```
Comment (ambarkhan) [inline Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs:234]: Codacy found an issue: [Remove this unnecessary cast to 'VariableSelectionList'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

File: Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs [hunk 2]
```diff
@@ -219,151 +219,21 @@ public static BusinessFlow ConvertQtestTestSuiteToBF(QtestTestSuite testSuite)
...
-                            else
-                            {
-                                stepActivityVar.LinkedVariableName = string.Empty; // clear old links
-                            }
-                        }
+                            stepActivityVar = new VariableSelectionList();
+                            stepActivityVar.Name = step.Params.Name;
+                            ObservableList<OptionalValue> optionalValuesList = new ObservableList<OptionalValue>();
+                            step.Params.Values.ForEach(z => optionalValuesList.Add(new OptionalValue(z)));
+                            ((VariableSelectionList)stepActivityVar).OptionalValuesList = optionalValuesList;
```
Comment (ambarkhan) [inline Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs:233]: Codacy found an issue: [Remove this unnecessary cast to 'VariableSelectionList'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5794604&bid=18966742)

**Review by JinendraGhodke — APPROVED** [review]

**Pull request merged**